### PR TITLE
WIP, adding skip links.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

This is the PR I worked on today in the live stream with @cmgorton and the stream chat on raise.dev, https://www.twitch.tv/videos/762574120.

A skip link currently only exists on the main page of a Forem instance.

## Related Tickets & Documents

* #1153
* [Skip Navigation Links](https://webaim.org/techniques/skipnav/)

## QA Instructions, Screenshots, Recordings

_Please replace this line with instructions on how to test your changes, as well
as any relevant images for UI changes._

## Added tests?

- [ ] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.forem.com
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

Billie Eilish and her brother doing funny dances

### CODE REVIEW

File: app/javascript/readingList/readingList.jsx [hunk 1]
```diff
@@ -214,7 +214,7 @@ export class ReadingList extends Component {
             onClick={this.toggleTag}
           />
 
-          <main className="crayons-layout__content">
+          <main id="articles-list" className="crayons-layout__content">
```
Comment (nickytonline) [inline app/javascript/readingList/readingList.jsx:217]: I will be changing the `id` attribute for all `<main />` elements in the site to something more generic as the only existing skip link uses an `id` attribute that is specific to the main feed on the home page of a Forem instance.

File: app/assets/stylesheets/moderators.scss [hunk 2]
```diff
@@ -122,7 +122,7 @@
   min-height: initial;
```
Comment (nickytonline) [inline app/assets/stylesheets/moderators.scss:122]: // TODO:  Ensure skip link on editor is above Forem banners.

**Comment by colabottles** [comment]
I'll take a crack at this if you want to assign this to me @nickytonline

**Comment by nickytonline** [comment]
All yours @colabottles! Thanks again for watching the stream today and thanks for the accessibility knowledge drops in the chat. 😎

**Comment by nickytonline** [comment]
I can assign the issue to you as well. You just need to comment on #1153  and then the issue can be assigned to you.

**Comment by colabottles** [comment]
> I can assign the issue to you as well. You just need to comment on #1153 and then the issue can be assigned to you.

Done. 🙂

**Comment by nickytonline** [comment]
Closing this PR as @colabottles is taking #1153 over.

**Pull request closed without merging**